### PR TITLE
0.67 Changelog

### PR DESCRIPTION
**Pull request by lewri**
New changelog. There's a TODO for packager notes I need Cyco to advise on ; and what #2191 lets someone do (if anything).

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,55 @@
...
+* Handymen will no longer try to repair machinery while a patient is using it
+* Psychiatrist couch has less chance of causing blocked paths
+* Roujin’s challenge is now properly applied (read: harder)
+* Serious Radiation is no longer contagious
+* Room blueprints can no longer go beyond external walls
+* Fixed incorrect button sounds in the Staff Management window
+* Fixed back button behaviour in Font Picker
+* Fixed a rare case where the next VIP date may be scheduled in the past
+* Fixed a rare case where awarded research points could be negative
+* Fixed a rare case of the balance sheet Hulking out
```
Comment (lewri) [inline changelog.txt:48]: There is a fallback for this line if we can't use the word Hulk

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,55 @@
+-------------------------------------------------------------------------------
+Version 0.67 - released xxx 2022
+-------------------------------------------------------------------------------
+# New Features/Enhancements
+* The backend code for MIDI playback has been changed and real pause logic has been
```
Comment (TheCycoONE) [inline changelog.txt:5]: This is windows only.

File: changelog.txt [hunk 3]
```diff
@@ -1,3 +1,55 @@
...
+  may change depending on your configuration from ≤0.66
+* The original demo gameplay video now plays when the CorsixTH main menu is idle (this
+  can be disabled in the configuration file)
+* File browser menus now have an OK button (at last!)
+* Selecting an existing file in the Save Game/Map browsers will now update the text
+  input box to that file name
+
+# Changes
+* Handymen now always stay in their assigned plots, except when going to rest
+* Staff members now have a maximum salary, which will be configurable in custom levels
```
Comment (TheCycoONE) [inline changelog.txt:16]: will be configurable or can be configured? would rather not comment on future features here.

File: changelog.txt [hunk 4]
```diff
@@ -1,3 +1,55 @@
...
+  revealed
+* Balance changes to drug improvement research
+* Check for updates can now be disabled in Settings
+* “Pause” now shows up when the “Build while pause” setting is on and the game is paused.
+* The gamelog file now includes a timestamp
+* Many, many, refactors of the code to make it better to understand for future developers
+
+# Translations
+* Chinese Traditional and Simplified translations have been updated. Thanks @lokpro
+* Russian has been updated. Thanks Matt1
```
Comment (TheCycoONE) [inline changelog.txt:30]: Why @ for lokpro but not Matt1?

File: changelog.txt (hunk 1 above)
Comment (TheCycoONE) [reply changelog.txt:48]: cute but turning green is much clearer

**Comment by TheCycoONE** [comment]
Thanks a lot @lewri - could you replace the smart quotes with regular ASCII characters `'` and `"`. Avoid any potential encoding issues.

File: changelog.txt (hunk 1 above)
Comment (Alberth289346) [reply changelog.txt:48]: I agree, I had no idea what it meant until I read the comment by TheCycoONE

File: changelog.txt [hunk 5]
```diff
@@ -1,3 +1,55 @@
+-------------------------------------------------------------------------------
+Version 0.67 - released xxx 2022
```
Comment (Alberth289346) [inline changelog.txt:2]: Don't forget to change the year as well :)

File: changelog.txt [hunk 6]
```diff
@@ -1,3 +1,55 @@
...
+* Balance changes to drug improvement research
+* Check for updates can now be disabled in Settings
+* “Pause” now shows up when the “Build while pause” setting is on and the game is paused.
+* The gamelog file now includes a timestamp
+* Many, many, refactors of the code to make it better to understand for future developers
+
+# Translations
+* Chinese Traditional and Simplified translations have been updated. Thanks @lokpro
+* Russian has been updated. Thanks Matt1
+* Fixed a bug where CJK languages could crash the game
```
[the remark this answers is not in the document]
Comment (Alberth289346) [reply changelog.txt:31]: words longer than the line length ?

Could refer to CJK languages as an example where this happens?

File: changelog.txt [hunk 7]
```diff
@@ -1,3 +1,55 @@
...
+* Contact methods to reach the CorsixTH Team have been updated
+
+Bug Fixes
+* Fixed an instance where objects could disappear from the map
+* Fixed a bug where the empty action queue message was prevented from happening
+* Fixed a bug where dying patients became undead and would have a habit of ruining everything 
+* Fixed broken dynamic info during epidemics
+* Handymen will no longer try to repair machinery while a patient is using it
```
Comment (Alberth289346) [inline changelog.txt:39]: Some listed bugs here "fix a bug" while others describe the changed behavior.

File: changelog.txt (hunk 4 above)
Comment (lewri) [reply changelog.txt:30]: I don't believe Matt1 has a github account. That is just their Discord username. I can add Discord in brackets if needed.

File: changelog.txt (hunk 7 above)
Comment (lewri) [reply changelog.txt:39]: It's never an exact science to be honest, I just group them as best as I can. E.g the handymen one is both a change and a bug fix (from a logic perspective).  I usually take the bugs to be 'correcting an undesirable/unwanted outcome' for the purposes of this. 

(Under that guise I may have to move the Epidemic one from changes down to bugfix)

Comment (lewri) [reply changelog.txt:38]: I should also moved the 'fixed a bug' ones down with the others of the same prefix.

File: changelog.txt (hunk 6 above)
[the remark this answers is not in the document]
Comment (lewri) [reply changelog.txt]: My bad, this can happen in any language (I checked lokpro's notes)

File: changelog.txt (hunk 7 above)
Comment (Alberth289346) [reply changelog.txt:38]: I just wasn't sure what the distinction was, I mean, you can easily rephrase the epidemics line to "Dynamic info now works during epidemics" or vice versa. But it's fine.

File: changelog.txt (hunk 4 above)
Comment (TheCycoONE) [reply changelog.txt:31]: No it's fine. Thanks.

**Comment by TheCycoONE** [comment]
LGTM

File: changelog.txt [hunk 8]
```diff
@@ -1,3 +1,55 @@
...
+* Room blueprints can no longer go beyond external walls
+* Fixed incorrect button sounds in the Staff Management window
+* Fixed back button behaviour in Font Picker
+* Fixed a bug where word-wrapping text on dialogs could crash the game
+* Fixed a rare case where the next VIP date may be scheduled in the past
+* Fixed a rare case where awarded research points could be negative
+* Fixed a rare case of the balance sheet being victim of a green highlighter
+
+# Packager Notes
+* Option added to build CLI tools for RNC decoding
```
Comment (lewri) [inline changelog.txt:51]: @TheCycoONE Did you stil want this line chaged to add in brackets "(not enabled by default)" or something?

Comment (TheCycoONE) [reply changelog.txt:51]: no

**Comment by TheCycoONE** [comment]
if this wasn't marked RFC I'd have already merged it. Merge when ready.

File: changelog.txt [hunk 9]
```diff
@@ -1,3 +1,55 @@
+-------------------------------------------------------------------------------
+Version 0.67 - released xxx 2023
```
Comment (lewri) [inline changelog.txt:2]: Update month on release

**Comment by lewri** [comment]
> if this wasn't marked RFC I'd have already merged it. Merge when ready.

Marked ready

**Pull request merged**